Parsers/ASimAuthentication/Tests/Salesforce_ASimAuthentication_DataTest.csv

```diff
@@ -0,0 +1,24 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 10 records (33.33%) for field [ActorUserIdType] of type [Enumerated]: [""SaleforceId""] (Schema:Authentication)"
```

Comment:
Please add this value to the test case and documentation

Reply:
@anki-narravula has been added already in tester.csv
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/291f58ec-8cfe-40d0-99d3-85b8f071a54b)